mackup/mackup.py

```diff
@@ -19,9 +19,9 @@ class Mackup(object):
 
     """Main Mackup class."""
 
-    def __init__(self):
+    def __init__(self, filename=None):
```

Comment:
Can you assert that the filename parameter is valid? e.g. https://github.com/lra/mackup/blob/master/mackup/config.py#L40